libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -771,7 +778,7 @@ function(add_libc_hermetic test_name)
                    ${fq_deps_list})
   # TODO: currently the dependency chain is broken such that getauxval cannot properly
   # propagate to hermetic tests. This is a temporary workaround.
-  if (LIBC_TARGET_ARCHITECTURE_IS_AARCH64)
+  if (LIBC_TARGET_ARCHITECTURE_IS_AARCH64 AND NOT LIBC_TARGET_OS_IS_UEFI)
```

Comment:
There should be a `TODO` comment explaining why this is disabled for UEFI.

Reply:
TODO shouldn't be needed since there's no AUX vector in UEFI.